### ISSUE TITLE
Search Widget/Popup: Description and Rating [display]

### ISSUE THREAD

**Issue by FadedOut** [Smile-SA/elasticsuite#2957]
**Is your question related to a problem? Please describe.**
This is probably not a "problem" or "bug" but just me missing something. If someone can point me to where I can enable the option to show the **product short description** and the **product rating** in the search widget/popup results that would be super and much appreciated.

**Additional context**
The results widget/popup here (using Luma theme/Smile 2.11.2/Magento 2.4.6):
![smile-elasticsearch-2-11-2-search-results-luma](https://github.com/Smile-SA/elasticsuite/assets/12685113/963396de-e7c6-4670-bf48-b8ff72507831)

Also, I found from a few years ago that it seems this was added (https://github.com/Smile-SA/elasticsuite/pull/1485), which is why I feel I'm probably just missing the setting for it. I've looked all over but there are many settings and it might just be somewhere I haven't looked. I also tried the Wiki but didn't find there either.

Thanks in advance!

**Comment by vahonc** [comment]
Hello @FadedOut,

As I can see PR [#1485](https://github.com/Smile-SA/elasticsuite/pull/1485) was closed and not merged into our module.

There is no simple way like just enabling in Magento BO some option to show the product short description or the product rating or any other additional product info in the Autocomplete (search widget/popup) results.

But you can do it via code with additional customization. For this take some inspiration from these two issues [#953](https://github.com/Smile-SA/elasticsuite/issues/953) and [#1416](https://github.com/Smile-SA/elasticsuite/issues/1416).
It should help to solve your problem.

BR,
Vadym

**Comment by FadedOut** [comment]
Hi @vahonc 

Thank you for getting back to me I much appreciate it.

I did not realize that PR was not actually merged. I just did some searching and found that. But the other two you linked me to I did not find. I will see if I can make sense of them, thanks.

I would just like to ask if it would be possible to just integrate those features right in - it seems this has been asked several times before so it's not a one-off request. It would make sense to be able to have more info there and select what shows. I request this because something like [MageWorx Search Autocomplete](https://github.com/mageworx/search-suite-autocomplete) does it (where I get the idea from)

![mageworx-search-autocomplete-product-field-options](https://github.com/Smile-SA/elasticsuite/assets/12685113/94635405-4f68-43f3-bd04-f747c1ce3b20)

it's just of course all the other features of this extension make this one better, but would be super nice to have the search widget/popup have options of what shows.

Thanks again for your time.
